src/Infrastructure/HealthChecks/ServiceBusSendHealthCheck.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Azure.Identity;
+using Azure.Messaging.ServiceBus;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Logging;
+using NServiceBus;
+using System.Diagnostics.CodeAnalysis;
+
+namespace SFA.DAS.Funding.ApprenticeshipEarnings.Infrastructure.HealthChecks;
+
+[ExcludeFromCodeCoverage]
+public class ServiceBusSendHealthCheck : BaseHealthCheck<ServiceBusSendHealthCheck>
```

Comment:
Should the send nomenclature in this change be publish? The actual check is doing a publish.